### DISCUSSION TITLE
LittleFS file retrieval

### DISCUSSION THREAD

**Discussion by czarnikjak** [micropython/micropython#13143] [Using MicroPython]
I accidentally removed all files from a folder on my Pi Pico with os.remove 🤦‍♂️
FS is LittleFS, is there a way to retrieve these files? I have not written to the flash since then, so they shouldn't be overwritten by anything.

**Comment by scruss** [comment]
This thread from the Raspberry Pi forum seemed to help people recover data [before](https://github.com/orgs/micropython/discussions/10923): [Recovering files from a Pico file system - Raspberry Pi Forums](https://forums.raspberrypi.com/viewtopic.php?t=337154)
